src/commands/custom/teaminfo.js

```diff
@@ -0,0 +1,56 @@
+const { SlashCommandBuilder } = require("discord.js");
+const { createEmbed } = require("../../utils/embedBuilder");
+
+module.exports = {
+  category: "custom",
+  data: new SlashCommandBuilder()
+    .setName("teaminfo")
+    .setDescription("Shows who is part of a team")
+    .addRoleOption((option) =>
+      option
+        .setName("team")
+        .setDescription("Select the team role")
+        .setRequired(true)
+    ),
+  async execute(interaction) {
+    await interaction.deferReply();
+    const selectedTeamRole = interaction.options.getRole("team");
+
+    if (!selectedTeamRole.name.includes("|")) {
+      interaction.editReply({
+        content: "Not a team role sorry!",
+        ephemeral: true,
+      });
+    } else {
+      const roleMembers = interaction.guild.members.cache.filter((member) =>
```

Comment:
adding `interaction.guild.members.cache.fetch()` before this line will ensure that the cache doesn't trip and give you only the user who inputted the command